projects/ngx-formentry/src/form-entry/form-renderer/form-renderer.component.scss

```diff
@@ -23,15 +23,15 @@
 
 .nav-link-text {
   margin-top: 0.125rem;
-  white-space: nowrap;
   overflow: hidden;
   text-overflow: ellipsis;
   max-width: calc(100% - #{spacing.$spacing-02});
-  @include type.type-style('heading-03');
+  @include type.type-style('label-01');
 }
 
 .nav-label {
   @include type.type-style('label-01');
+  color: #696969;
```

Comment:
#525252 is the default carbon hex for input labels. It's probably a better fit here for consistency:

![CleanShot 2024-03-28 at 1  03 43@2x](https://github.com/openmrs/openmrs-ngx-formentry/assets/8509731/60f7f18b-42e9-421e-8166-4a9d443c91c7)

Also, when we migrate the form engine to use [carbon colour tokens](https://carbon-elements.netlify.app/colors/examples/preview/), we could swap it out for `gray-70`.

It also gets a better contrast score than #696969:

![CleanShot 2024-03-28 at 1  06 37@2x](https://github.com/openmrs/openmrs-ngx-formentry/assets/8509731/97cc63d6-6cb1-4499-b5b2-ddf460bd38e4)

![CleanShot 2024-03-28 at 1  08 04@2x](https://github.com/openmrs/openmrs-ngx-formentry/assets/8509731/5d8102f3-6f21-4dd4-9cc4-263f56ef0b34)

Reply:
Agreed. Thanks. @denniskigen